src/Widgets/Dropdown.jsx

```diff
@@ -4,38 +4,35 @@ class DropDown extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      val: '', // ddDefault
+      ddVal: '',
     };
-    // method bindings
     this.handleChange = this.handleChange.bind(this);
   }
 
-  // methods
-  handleChange(e) { this.setState({val: e.target.value}); }
+  handleChange(e) {
+    this.setState({ddVal: e.target.value});
+    this.props.ddCB(e.target.value);
+  }
 
   handleSubmit(e) { e.preventDefault(); }
 
   render() {
-    // js;
-    // the following should be unique to the specific dropdown -
-    // for=
-    // name=
-    // id= ddid = "dropdown-reviews"
-    // default sort? for reviews: !relevant!
+    // ddValue is "Default value" / initialization
     let ddLabel = this.props.ddLabel;
     let ddList = this.props.ddList;
-    let ddDefault = this.props.ddDefault;
+    let ddVal = this.props.ddVal;
```

Comment:
give these better names when you get a chance. variable names are not easy to interpret for reader